src/ios/Calendar.m

```diff
@@ -392,8 +392,12 @@ - (NSMutableArray*) eventsToDataArray: (NSArray*)matchingEvents {
     if (event.location != nil) {
       [entry setObject:event.location forKey:@"location"];
     }
-    if (event.notes != nil) {
-      [entry setObject:event.notes forKey:@"message"];
+    if (event.notes != nil || event.URL != nil) {
+      NSString *notes = event.notes ?: @"";
+      NSString *url = event.URL.absoluteString ?: @"";
```

Comment:
`event.URL` can be nil at this stage.

Reply:
I believe that is taken care with the `if` encapsulating this code.

Reply:
No, because it's an `or`.

Reply:
Yes, but in that case the `event.notes` can be nil as well right?

Reply:
Or is it because I'm doing `event.URL.absoluteString` ?

Reply:
Yes, and that's ok since we're not calling anything in it. The URL case we're calling absoluteString in it and it will crash if event.URL is nil.

Reply:
Yes, makes sense, will update the PR to verify the `event.URL` and only then call the `absoluteString`

Reply:
Actually I think this was never a problem.
I believe the URL it's a `NSURL`. 
So in this case if `URL` property is `nil` then it will never crash, obj-c will just return nil.
One of Objective-C messaging caveats. If you try to access the method or property of a nil NSObject, it will return nil.

Reply:
I think that's true if you use the selector notation, but not by accessing properties directly with `.`. Something like `[event.URL absoluteString]` might work but I'm not 100% sure.

Reply:
It was quite a long time ago and I needed to be sure because I had the same doubt as you. But I made a small test and it works even accessing with directly with `.`.